admin-dev/themes/new-theme/webpack.config.js

```diff
@@ -81,19 +81,19 @@ module.exports = {
   },
   plugins: [
     new ExtractTextPlugin('theme.css'),
-    new webpack.optimize.UglifyJsPlugin({
-      sourceMap: false,
-      compress: {
-        sequences: true,
-        conditionals: true,
-        booleans: true,
-        if_return: true,
-        join_vars: true,
-        drop_console: true
-      },
-      output: {
-        comments: false
-      }
-    })
+    // new webpack.optimize.UglifyJsPlugin({
```

Comment:
well , I understand why you did it.

More than ever, we still need a development configuration for webpack